package.json

```diff
@@ -12,7 +12,9 @@
     "url": "https://github.com/JedWatson/react-select.git"
   },
   "dependencies": {
+    "@emotion/memoize": "^0.7.1",
     "classnames": "^2.2.5",
+    "create-emotion": "^10.0.4",
     "emotion": "^9.1.2",
```

Comment:
We might not have a direct dependency on emotion anymore.

Reply:
Yeah, this should be removed